grants_tagger/preprocess_mesh.py

```diff
@@ -1,13 +1,20 @@
 """
 Preprocess JSON Mesh data from BioASQ to JSONL
 """
+from configparser import ConfigParser
 from pathlib import Path
 import argparse
 import json
+import os
+
+import pandas as pd
 
 def yield_raw_data(input_path):
     with open(input_path, encoding='latin-1') as f_i:
-        for line in f_i:
+        for i, line in enumerate(f_i):
+            if i == 0:
```

Comment:
Perhaps `f_i.readline()` before the loop is cleaner?

Reply:
good idea

Reply:
Resolved by 95920652fba6f9458ca653d401501a62404bcc99